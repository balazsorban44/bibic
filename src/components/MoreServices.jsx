import React, {Component} from 'react'
import Gallery from './shared/Gallery'


export default class MoreServices extends Component {

  componentDidMount() {
    window.scrollTo(0, 0)
  }

  render() {
    return(
      <section id="szolgaltatasok">
        <h2>Szolgáltatásaink</h2>
        <Gallery path="szolgaltatasaink"/>
      </section>
    )}
}