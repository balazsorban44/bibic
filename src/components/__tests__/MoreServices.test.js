import MoreServices from "../MoreServices"

global.scrollTo = jest.fn()

describe("MoreServices component", () => {
  const wrapper = shallow(<MoreServices/>)

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot()
  })

  it("scrolls to top on mount", () => {
    expect(scrollTo).toBeCalled()
  })

})
