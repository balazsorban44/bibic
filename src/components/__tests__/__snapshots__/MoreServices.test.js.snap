// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoreServices component renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MoreServices />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          Szolgáltatásaink
        </h2>,
        <Component
          path="szolgaltatasaink"
        />,
      ],
      "id": "szolgaltatasok",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Szolgáltatásaink",
        },
        "ref": null,
        "rendered": "Szolgáltatásaink",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "path": "szolgaltatasaink",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            Szolgáltatásaink
          </h2>,
          <Component
            path="szolgaltatasaink"
          />,
        ],
        "id": "szolgaltatasok",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Szolgáltatásaink",
          },
          "ref": null,
          "rendered": "Szolgáltatásaink",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "path": "szolgaltatasaink",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
